### ISSUE TITLE
GoTo doesn't work on packages inside my repository

### ISSUE THREAD

**Issue by nikopatrik** [neoclide/coc.nvim#1740]
## versions

vim version: VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Feb 17 2020 07:45:29)
node version: v13.12.0
coc.nvim version: 0.0.77-817efd5108
term: iTerm.app
platform: darwin

## Output channel: Python

Starting Jedi Python language engine.
##########Linting Output - pylint##########

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

## Describe the bug
coc-definition command doesn't work on project packages, but on site-packages it work well. Completions don't work too.

## Reproduce the bug
Node installed by calling brew install node. Using provided minimal vimrc below install vundle and coc.nvim.  When coc.nvim installed,go to ~/.vim/bundle/coc.nvim where coc is installed and checkout to release branch. Inside vim then call CocInstall coc-python, CocCommand python.setInterpret I set interpret to venv python, which I have created using python from pyenv. Inside venv is installed Jedi version  0.16.0. 
Project Im working on:
[diffkemp](https://github.com/viktormalik/diffkemp)
PYTHONPATH contains root directory of the project.
Problem -> when I open file in project diffkemp/diffkemp.py, I can use goto command for ArgParser and other python packages but cant be used for eg. KernelSource which is class defined inside this project.

```vim
set nocompatible
filetype off
set rtp+=~/.vim/bundle/Vundle.vim
call vundle#begin()
Plugin 'VundleVim/Vundle.vim'
Plugin 'neoclide/coc.nvim'
call vundle#end()            " required
filetype plugin indent on    " required
nmap <silent> gd <Plug>(coc-definition)
nmap <silent> gi <Plug>(coc-implementation)
```

**Comment by chemzqm** [comment]
Should be problem of your language server, checkout https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel

**Comment by nikopatrik** [comment]
Somehow I managed to find out that it works in other files except this because its called  same as package, I tried to rename it and it started to work. I dont know why it doesn't work because in same project is same case where simpll package contains simpll.py file a in that file it works. By debuginng LS a I haven't found anything because it log only when GoTo works but no log have been print when it didnt work.
